modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms/smt-schemaregistryreplicator.adoc

```diff
@@ -0,0 +1,16 @@
+= SchemaRegistryReplicator (Redpanda)
+:description: How to set SchemaRegistryReplicator (Redpanda) Single Message Transform for a connector.
+:page-cloud: true
+
+The SchemaRegistryReplicator (Redpanda) is a transform to filter and process Redpanda Schema Registry messages.
```

Comment:
I would rather say to `replicate` instead of `filter and process`, as this is its main purpose. Filtering is an implementation detail.